README.md

```diff
@@ -12,26 +12,10 @@ For Spark 1.x support, please check [spark1.x](https://github.com/springml/spark
 ## Linking
 You can link against this library in your program at the following ways:
 
-### Maven Dependency
```

Comment:
Shall we make Databricks-specific README changes in a separate PR and in a separate file? This will help us contribute the changes to upstream.